### DISCUSSION TITLE
Help with ST7789

### DISCUSSION THREAD

**Discussion by ege77** [Bodmer/TFT_eSPI#2338] [Q&A - General] [answered]
Hello . I couldn't use my st7789 1.69 screen with TFT_eSPI. I have been trying to solve this problem for a long time. Please show me which files in the TFT_eSPI folder I should replace!

**Comment by Bodmer** [comment, marked as answer]
Edit the User_Setup_Select.h to point to a suitable setup file for the processor and interface type you are using. There are online tutorials that a Google search should find.
